### PR TITLE
skopeo: update to 1.9.2

### PR DESCRIPTION
**Pull request by CameronNemo**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by CameronNemo** [comment]
@paper42 Can you retrigger CI? Looks like the repos were acting up or something. xlint and most of these builds ran successfully locally. (I did not try i686 locally but I do not see why it would fail.)

**Comment by CameronNemo** [comment]
nvmd rebased and force pushed.

File: srcpkgs/skopeo/template [hunk 1]
```diff
@@ -14,16 +15,26 @@ maintainer="Cameron Nemo <cnemo@tutanota.com>"
...
 distfiles="https://github.com/containers/${pkgname}/archive/v${version}.tar.gz"
-checksum=453bdcce16767696ed71046b60ad7b34358b183b50eb5aa708ced0b5ea2927b1
+checksum=9a321ba75f213e5c46cba7f92073c2437137a56d3140c9ab6e723fb92890f9d0
 make_dirs="/var/lib/atomic/sigstore 0755 root root"
+_completions="bash zsh fish"
 
 post_build() {
 	make docs
+	local cli=$(find $GOPATH/bin -name skopeo)
+	echo $cli
```
Comment (paper42) [inline srcpkgs/skopeo/template:25]: I think you forgot to drop this

File: srcpkgs/skopeo/template [hunk 2]
```diff
@@ -14,16 +15,26 @@ maintainer="Cameron Nemo <cnemo@tutanota.com>"
...
 homepage="https://github.com/containers/skopeo"
 distfiles="https://github.com/containers/${pkgname}/archive/v${version}.tar.gz"
-checksum=453bdcce16767696ed71046b60ad7b34358b183b50eb5aa708ced0b5ea2927b1
+checksum=9a321ba75f213e5c46cba7f92073c2437137a56d3140c9ab6e723fb92890f9d0
 make_dirs="/var/lib/atomic/sigstore 0755 root root"
+_completions="bash zsh fish"
 
 post_build() {
 	make docs
+	local cli=$(find $GOPATH/bin -name skopeo)
```
Comment (paper42) [inline srcpkgs/skopeo/template:24]: isn't this the same path every time?

Comment (CameronNemo) [reply srcpkgs/skopeo/template:24]: I got this from the k3d template, but it works fine if I just replace `$cli` with `$GOPATH/bin/skopeo`.

File: srcpkgs/skopeo/template (hunk 1 above)
Comment (CameronNemo) [reply srcpkgs/skopeo/template]: yep

File: srcpkgs/skopeo/template (hunk 2 above)
Comment (CameronNemo) [reply srcpkgs/skopeo/template]: alright I see why k3d uses find... differs based on architecture/native/cross.

**Pull request merged**